ceph.spec.in

```diff
@@ -804,7 +804,6 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %post
```

Comment:
We should probably just rename the (unnamed) `%post` / `%preun` / `%postun` sections to `%post base`, etc